src/abstract/interfaces/IBatchGovernor.sol

```diff
@@ -100,7 +100,7 @@ interface IBatchGovernor is IGovernor {
      * @param  v           An ECDSA secp256k1 signature parameter.
      * @param  r           An ECDSA secp256k1 signature parameter.
      * @param  s           An ECDSA secp256k1 signature parameter.
-     * @return weight      The number of votes cast for each proposal (the same for all of them).
+     * @return weight      The number of votes casted for each proposal (the same for all of them).
```

Comment:
Unfortunately, cast is one of those words that remains that way in past tense.

Reply:
Ah yes, forgot we kept `IGovernor` this way to stay compatible with OZ Governor.
Only `the` to `to`: https://github.com/MZero-Labs/ttg/pull/277/commits/0d750db0b6a714f17e24a15ae7be00a2307b5137